visualization/include/pcl/visualization/registration_visualizer.h

```diff
@@ -157,10 +157,7 @@ namespace pcl
       inline std::string
       getIndexedName (std::string &root_name, std::size_t &id)
```

Comment:
Can we increase the const guarantee (some other PR)
```suggestion
      getIndexedName (const std::string &root_name, std::size_t &id) const
```